os/gcron/gcron.go

```diff
@@ -120,3 +120,8 @@ func Start(name ...string) {
 func Stop(name ...string) {
 	defaultCron.Stop(name...)
 }
+
+// GracefulStop Blocks and waits all current running jobs done.
+func GracefulStop() {
```

Comment:
Would it better use `StopGracefully` instead of `GracefulStop`, as the naming `GracefulStop` is a little bit chinglish?